diagnostic.md

```diff
@@ -28,25 +28,25 @@ this repository (`unix-cli-diagnostic`).
 5. Navigate back up to `cli-diagnostic`, and delete the `temp` directory (with `temp.md` inside of it).
 Use `ls` to show the contents of `cli-diagnostic` - was `temp` deleted?
 
+Yes, it was deleted using rm -rf
+
 ## Absolute and Relative Paths
 
 Open up this file in Atom, and write your answers below (where indicated).
 
 1. Is `/Users/blah_blah/Desktop` a relative path or an absolute path? How do you know?
 
- <!-- Answer Starts Here -->
-
- <!-- Answer Ends Here -->
+ <!--It is an absolute path because it is telling us we are looking for our root directory followed by home directory then our desktop->
 
 2. Suppose that we're working on a project, and we want to use a font that's being hosted somewhere on the internet. Would we use an absolute or relative path to refer to it? Why?
 
- <!-- Answer Starts Here -->
+ <!-- We would use a relztive path because it is not in our computer but on the internet.-->
```

Comment:
It would be an absolute path because you need the entire path from the internet to access the information.